### ISSUE TITLE
feat: Improve Dark mode toggle button

### ISSUE THREAD

**Issue by iamkryptonite** [bench-routes/bench-routes#375]
**Is your feature request related to a problem? Please describe.**
The current switch button for toggling the theme is not very good looking.As a new user, just by 
looking at it we cannot determine its function until we hover over the button and see the tooltip.

**Describe the solution you'd like**
Instead of just the plain switch button we can implement something like this.
![Redux_A_predictable_state_container_for_JavaScript_apps_Redux](https://user-images.githubusercontent.com/43510609/111423894-f686ed00-8716-11eb-8b71-82755c3f374a.png)

![Redux_A_predictable_state_container_for_JavaScript_apps_Redux](https://user-images.githubusercontent.com/43510609/111424018-2df59980-8717-11eb-897e-97b29c6a6a35.png)

**Additional Context**
I would like to work on this.

**Comment by iamkryptonite** [comment]
@muskankhedia if this is approved I would like to work on this.

**Comment by muskankhedia** [comment]
@iamkryptonite Sure, go ahead.

**Comment by iamkryptonite** [comment]
Can we use libraries other than material-ui?

> @iamkryptonite Sure, go ahead.

**Comment by Harkishen-Singh** [comment]
@iamkryptonite yes you can, but they should match the overall theme and at the same time, not be too heavy.
